### PR TITLE
chore: simplify exporter

### PR DESCRIPTION
**Pull request by syunull**
[Ticket](https://sironamedical.atlassian.net/browse/PE-2326)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,19 +3,18 @@ module sironamedical/vicimonitor
 go 1.21.0
 
 require (
-	github.com/mitchellh/go-ps v1.0.0
-	github.com/prometheus/client_golang v1.17.0
+	github.com/prometheus/client_golang v1.20.2
```
Comment (amegianeg) [inline go.mod:6]: Isn't the latest 1.23.0? or previous 1.22.6 if the latest may not be stable-enough.

File: main.go [hunk 2]
```diff
@@ -1,65 +1,87 @@
...
+
+	cancel()
+	wg.Wait()
+}
+
+func collectMetrics(ctx context.Context, session *vici.Session, interval time.Duration) {
+	ticker := time.NewTicker(interval * time.Second)
+	defer ticker.Stop()
+
+	monitor := metrics.NewPrometheus(session)
```
Comment (amegianeg) [inline main.go:77]: Does this mean we are considering go back to Prometheus? 😃

**Review by amegianeg — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (syunull) [reply go.mod:6]: where do you see version 1.23? 

https://github.com/prometheus/client_golang/releases

File: main.go (hunk 2 above)
Comment (syunull) [reply main.go:77]: datadog supports collecting metrics from openmetric endpoints, every metric is just a custom metric though

File: go.mod (hunk 1 above)
Comment (amegianeg) [reply go.mod:6]: Wait, I read this as the Golang version 🤦 

nvm, sorry for the noise!

**Pull request merged**